tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/parser/GradleDependencyUpdater.java

```diff
@@ -37,19 +38,27 @@ public GradleDependencyUpdater( String scriptContents ) throws MultipleCompilati
         nodes = builder.buildFromString( scriptContents );
     }
 
+    public FindDependenciesVisitor insertDependency( GradleDependency gradleDependency ) throws IOException
```

Comment:
What happens if there is no "dependencies {}" block at all?  Does this work?  Remember the "dependencies" call could happen in a parent build.gradle.  Do we have units tests for "insertDependency" for when there is no "dependencies" code block at all?